src/common/rand/rand_nist.c

```diff
@@ -38,6 +38,7 @@ typedef struct {
 } AES256_CTR_DRBG_struct;
 
 static AES256_CTR_DRBG_struct DRBG_ctx;
+static AES256_CTR_DRBG_struct DRBG_ctx_backup;
```

Comment:
aha -- here's the "duplicate state" -- but again, it only holds a single "second" state, so at the very least it should be stated at the API that calling `save` twice may result in "surprising" behaviour.